File: messages/src/main/java/com/thundashop/core/accountingmanager/AccountingManagerConfig.java

package com.thundashop.core.accountingmanager;

import com.thundashop.core.common.DataCommon;
import java.util.ArrayList;
import java.util.HashMap;
import java.util.List;

public class AccountingManagerConfig extends DataCommon {
    
    
    public List<Integer> statesToInclude = new ArrayList();
    private HashMap<String, TransferFtpConfig> configrations = new HashMap();
    
    /* ftp accounting stuff */
    public String vendor = "svea";
    public String username = "";
    public String password = "";
    public String hostname = "";
    public String path = "";
    public String invoice_path = "";
    public String extension = "";
    public boolean useSftp = false;
    public boolean useActiveMode = false;
    public boolean transferAllUsersConnectedToOrders = false;
    public Integer port = 21;
    
    public TransferFtpConfig getCreditorConfig() {
        TransferFtpConfig config = configrations.get("creditor");
        if(config == null) {
            config = new TransferFtpConfig();
        }
        return config;
    }
    
    public TransferFtpConfig getBComConfig() {
        TransferFtpConfig config = configrations.get("bookingcomratemanager");
        if(config == null) {
            config = new TransferFtpConfig();
        }
        return config;
    }
    
}

File: html5apps/terminal/events/core/accountingmanager/AccountingManagerConfig.php
<?php
class core_accountingmanager_AccountingManagerConfig extends core_common_DataCommon  {
	/** @var String */
	public $statesToInclude;

	/** @var java.util.HashMap */
	public $configrations;

	/** @var String */
	public $vendor;

	/** @var String */
	public $username;

	/** @var String */
	public $password;

	/** @var String */
	public $hostname;

	/** @var String */
	public $path;

	/** @var String */
	public $invoice_path;

	/** @var String */
	public $extension;

	/** @var String */
	public $useSftp;

	/** @var String */
	public $useActiveMode;

	/** @var String */
	public $transferAllUsersConnectedToOrders;

	/** @var String */
	public $port;

}
?>